docs/designers-developers/designers/design-resources.md

```diff
@@ -1,6 +1,57 @@
 # Resources
 
-The [SketchPress](https://github.com/10up/SketchPress) project includes a library of Gutenberg design components helpful for designing and prototyping:
+## Figma
+
+[Figma](https://www.figma.com/) includes a library of [Gutenberg UI components](https://www.figma.com/file/ZtN5xslEVYgzU7Dd5CxgGZwq/WordPress-Components?node-id=491%3A3280) found in the `@wordpress/components` npm package, or “WordPress Components”. They are stable, fully supported, up to date, and ready for use in designs and prototypes. `@wordpress/components` is an npm package that is published via the Gutenberg repo. 
+
+### How to get access
+To join the WordPress.org Figma team, ping in #design on chat.wordpress.org and let us know whether you need view-only or edit access. A Figma admin will contact you for your email address to set up your account.
```

Comment:
There's some copy here that we may be able to borrow/duplicate: https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#how-can-designers-contribute

Reply:
Awesome, will take a look and add 👍 Thanks!

Reply:
Updated.